src/main/java/org/apache/commons/net/ftp/FTPFile.java

```diff
@@ -466,6 +467,9 @@ public String toFormattedString(final String timezone) {
 
     /**
      * Gets a string representation of the FTPFile information.
+     * The returned output will be an FTP server raw listing.
+     *
+     * @see #getRawListing()
      *
```

Comment:
No blank line between tags.

Reply:
done, I also removed the blank line in `toFormattedString` Javadoc